src/app/work-item/work-item-list/work-item-list.module.ts

```diff
@@ -12,6 +13,7 @@ import { WorkItemListEntryComponent } from './work-item-list-entry/work-item-lis
   imports:      [
     CommonModule, 
     WorkItemQuickAddModule,
+    Ng2Dropdown,
     DropdownModule,
```

Comment:
This DropdownModule the one I created, and Ng2Dropdown uses the same name as DropdownModule. For now I need both of them, so renamed one while importing up there.